test/cppsuite/test_harness/workload/thread_context.h

```diff
@@ -99,14 +100,15 @@ class transaction_context {
     int64_t _target_op_count = 0;
     bool _in_txn = false;
 
+    WT_SESSION *_session;
```

Comment:
Should we set it to nullptr as we did for `_timestamp_manager` ?

Reply:
Yes

Reply:
You can have a reference like: `scoped_cursor&`.

I don't think it's a big deal though.